### PR TITLE
Chore/daef 161 Send Daedalus and Cardano node logs to papertrail

### PR DESCRIPTION
**Pull request by darko-mijic**
This PR introduces tailing off `Daedalus.log` and `cardano-node.log` and sends new lines to papertrail.

### CODE REVIEW

File: electron/main.development.js [hunk 1]
```diff
@@ -1,16 +1,36 @@
...
+  remoteLog.info(line);
+});
+
+// Tail Cardano node log and send it to remote logging server
+const cardanoNodeLogFilePath = path.join(appLogFolderPath, 'cardano-node.log');
+
+const cardanoNodeLogTail = new Tail(cardanoNodeLogFilePath);
+
+cardanoNodeLogTail.on('line', (line) => {
+  remoteLog.info(line);
```
Comment (domenkozar) [inline electron/main.development.js]: Maybe we should prefix node here, just to be able to differentiate between the two.

File: electron/lib/remoteLog.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import Log from 'electron-log';
+import winston from 'winston';
+
+// Requiring `winston-papertrail` will expose
+// `winston.transports.Papertrail`
+require('winston-papertrail').Papertrail; // eslint-disable-line
+
+const winstonPapertrail = new winston.transports.Papertrail({
+  host: 'logs5.papertrailapp.com',
+  port: 43689
```
Comment (domenkozar) [inline electron/lib/remoteLog.js]: we should also set here `program` to differentiate between cardano-node and daedalus output.

Comment (darko-mijic) [inline electron/lib/remoteLog.js]: Yes. Done ;)

**Comment by domenkozar** [comment]
Looks good (besides from conflicts).

**Pull request merged**